lib/alchemy/resources_helper.rb

```diff
@@ -71,20 +71,18 @@ def resource_model
     #
     def render_attribute(resource, attribute, options = {})
       options.reverse_merge!(truncate: 50)
-      value = resource.send(attribute[:name])
-      if (relation = attribute[:relation]) && value.present?
-        record = relation[:model_association].klass.find(value)
-        value = record.send(relation[:attr_method])
-      elsif attribute[:type] == :datetime && value.present?
-        value = l(value)
+      if attribute[:relation]
+        record = resource.send(attribute[:relation][:name])
+        value = record.present? ? record.send(attribute[:relation][:attr_method]) : resource.send(attribute[:name])
```

Comment:
`record` might be `nil`

Before , the `rescue` below rendered a not found message, if this happens. We should keep this behavior.